### ISSUE TITLE
Missing files in sdist

### ISSUE THREAD

**Issue by thatch** [andbue/nashi#2]
It appears that the manifest is missing at least one file necessary to build
from the sdist for version 0.0.35. You're in good company, about 5% of other
projects updated in the last year are also missing files.

```
+ /tmp/venv/bin/pip3 wheel --no-binary nashi -w /tmp/ext nashi==0.0.35
Looking in indexes: http://10.10.0.139:9191/root/pypi/+simple/
Collecting nashi==0.0.35
  Downloading http://10.10.0.139:9191/root/pypi/%2Bf/13a/32a6295258b7c/nashi-0.0.35.tar.gz (46 kB)
    ERROR: Command errored out with exit status 1:
     command: /tmp/venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-wheel-egcssbf6/nashi/setup.py'"'"'; __file__='"'"'/tmp/pip-wheel-egcssbf6/nashi/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-wheel-egcssbf6/nashi/pip-egg-info
         cwd: /tmp/pip-wheel-egcssbf6/nashi/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-wheel-egcssbf6/nashi/setup.py", line 3, in <module>
        with open("../README.md") as f:
    FileNotFoundError: [Errno 2] No such file or directory: '../README.md'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by andbue** [comment]
Thanks, should be fixed now with nashi 0.0.36

**Comment by thatch** [comment]
Confirmed, works now!
